core/domain/training_job_result_domain.py

```diff
@@ -0,0 +1,81 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Functions and classdefs related to protobuf files used in Oppia-ml"""
+
+from core.classifiers import algorithm_registry
+from core.domain.proto import training_job_response_payload_pb2
+
+class TrainingJobResult(object):
+    """TrainingJobResult domain object.
+
+    This domain object stores the of training job result along with job_id and
+    algorithm_id. The training job result is the trained classifier data.
+    """
+
+    def __init__(self, job_id, algorithm_id, classifier_data):
+        """Initializes TrainingJobResult object.
+
+        Args:
+            job_id: str. The id of the training job whose results are stored
+                in classifier_data.
+            algorithm_id: str. The id of the algorithm of the training job.
+            classifier_data: object. Frozen model of the corresponding
+                training job.
+        """
+        self.job_id = job_id
+        self.algorithm_id = algorithm_id
+        self.classifier_data = classifier_data
+
+    def validate(self):
+        """Validate that TrainigJobResult object stores correct data.
+
+        Raises:
+            Exception: str. If the classifier data is stored in a field
```

Comment:
Drop the "If" -- just start with "The classifier ..."

Reply:
Done.